app/controllers/smart_proxies_controller.rb

```diff
@@ -71,7 +95,7 @@ def action_permission
     case params[:action]
       when 'refresh'
         :edit
-      when 'ping', 'version'
+      when 'ping', 'version', 'plugin_version', 'tftp_server'
```

Comment:
if this is going to be an infra for plugins to add, consider setting it up in a method or something